README.md

```diff
@@ -1,6 +1,8 @@
 
 # Probe Particle Model (PPM)
 
+[![Documentation Status](https://readthedocs.org/projects/probeparticlemodel/badge/?version=main)](https://probeparticlemodel.readthedocs.io/en/main/?badge=main)
```

Comment:
Do we want to have this documentation for the whole code, which is absolutely not like 1.st meeting tutorial that high. I would put it on a same the same level as the inner wiki?!

Reply:
Well, at least personally, I would slightly change the first sub-section of README called **References** to something like **Further information:** This one should have 3 points:
 - https://github.com/Probe-Particle/ProbeParticleModel#notable-publications-using-probe-particle-model Which points to literature at the end of README
 - https://probeparticlemodel.readthedocs.io/en/readthedocs/index.html 
 - https://github.com/Probe-Particle/ProbeParticleModel/wiki

Otherwise we have there the literature 2 times.

Reply:
Sounds good. I changed it to look like this: https://github.com/Probe-Particle/ProbeParticleModel/blob/readthedocs/README.md. Would that be okay?